config.json

```diff
@@ -18,7 +18,7 @@
   "RDMAPort"      : 1,
   "PerSandboxLog" : false,
   "ReserveCpuCount": 1,
-  "ShimMode"      : false,
```

Comment:
Please keep the default ShimMode as false

Reply:
done